src/Http/Controllers/Admin/Concerns/FormSubmitOptions.php

```diff
@@ -0,0 +1,254 @@
+<?php
+
+namespace A17\Twill\Http\Controllers\Admin\Concerns;
+
+use Illuminate\Database\Eloquent\Model;
+
+trait FormSubmitOptions
```

Comment:
Maybe we can rename this to "HasDefaultFormSubmitOptions"?

Reply:
This is a bit nitpick-y, but here are my reasons for keeping the curent name:

1. In Twill, I believe the `HasX` naming is typically used for model traits (aka. behaviors). I would prefer to have a different convention for controller traits, hence the `Concerns\X` structure.

2. `HasDefaultFormSubmitOptions` makes sense with regards to [#1719](https://github.com/area17/twill/pull/1719) but I think this PR goes a bit beyond the "defaults" by introducing the variations for the context of draft revisions.